modules/navbar.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable linebreak-style */
+
+export default function showBlock(e) {
+  const sectionList = document.getElementById('list');
+  const sectionAdd = document.getElementById('add_book');
+  const sectionContact = document.getElementById('contact');
+  switch (e) {
```

Comment:
Same here please replace the function with an arrow function. :+1: :smiley: :smiley: